lib/safer_rails_console/patches/boot/sandbox_flag.rb

```diff
@@ -20,6 +20,8 @@ def parse_arguments(arguments)
               OptionParser.new do |opt|
                 opt.banner = 'Usage: rails console [environment] [options]'
                 opt.on('-s', '--[no-]sandbox', 'Explicitly enable/disable sandbox mode.') { |v| options[:sandbox] = v }
+                opt.on('-w', '--writable', 'Alias for --no-sandbox.') { |v| options[:writable] = v }
```

Comment:
What happens if a user providers conflicting flags like `-w -r`?

Reply:
Only one takes into effect at once - the precedence is -s then -r then -w.

Reply:
That's probably fine since I don't think it's a big deal.  We could also error in this case.